src/ir/field_selector.cc

```diff
@@ -0,0 +1,13 @@
+#include "src/ir/field_selector.h"
+
+#include "absl/strings/str_cat.h"
+
+#include <string>
+
+namespace raksha::ir {
+
+std::string FieldSelector::ToString() const {
```

Comment:
Such simple functions could move into the header itself?

Reply:
Addressed in #68.